sql/mob_spawn_points.sql

```diff
@@ -6122,15 +6122,15 @@ INSERT INTO `mob_spawn_points` VALUES (16900320,'Shieldtrap','Shieldtrap',20,0,0
 -- ------------------------------------------------------------
 
 -- Ancient Vows
-INSERT INTO `mob_spawn_points` VALUES (16904193,'Mammet-19_Epsilon','Mammet-19 Epsilon',1,-607.789,81.699,-5.607,243);
-INSERT INTO `mob_spawn_points` VALUES (16904194,'Mammet-19_Epsilon','Mammet-19 Epsilon',1,-609.317,81.699,1.053,14);
-INSERT INTO `mob_spawn_points` VALUES (16904195,'Mammet-19_Epsilon','Mammet-19 Epsilon',1,-598.712,81.699,-1.929,94);
-INSERT INTO `mob_spawn_points` VALUES (16904196,'Mammet-19_Epsilon','Mammet-19 Epsilon',1,-4.2,1.534,-6.033,243);
-INSERT INTO `mob_spawn_points` VALUES (16904197,'Mammet-19_Epsilon','Mammet-19 Epsilon',1,-5.728,1.534,0.627,14);
-INSERT INTO `mob_spawn_points` VALUES (16904198,'Mammet-19_Epsilon','Mammet-19 Epsilon',1,4.877,1.534,-2.355,94);
-INSERT INTO `mob_spawn_points` VALUES (16904199,'Mammet-19_Epsilon','Mammet-19 Epsilon',1,592.961,-78.254,-5.177,243);
-INSERT INTO `mob_spawn_points` VALUES (16904200,'Mammet-19_Epsilon','Mammet-19 Epsilon',1,591.433,-78.254,1.483,14);
-INSERT INTO `mob_spawn_points` VALUES (16904201,'Mammet-19_Epsilon','Mammet-19 Epsilon',1,602.038,-78.254,-1.499,94);
+INSERT INTO `mob_spawn_points` VALUES (16904193,'Mammet-19_Epsilon','Mammet-19 Epsilon',1,-600.000,82.200,6.176,253);
```

Comment:
Are these values assumed?  They are very close, but don't appear to match the caps I have from this fight:
16904193 - -600.000,81.699,0.000
16904194 - -600.000,81.900,5.000
16904195 - -600.000,81.699,-5.000

Reply:
I did not see coordinates for them in the captures, I must of missed it. I took them from various videos and approximated the locations based on the video evidence i had. I can change them.